x/rewards/keeper/providers.go

```diff
@@ -31,6 +31,7 @@ func (k Keeper) distributeMonthlyBonusRewards(ctx sdk.Context) {
 	totalRewarded := sdk.ZeroInt()
 	// specs emissions from the total reward pool base on stake
 	specs := k.specEmissionParts(ctx)
+	bonusRewardsInfo := ""
```

Comment:
instead, aggregate to details the rewards where key is provider + chainID and value is rewards

Reply:
done